README.md

```diff
@@ -0,0 +1,26 @@
+# org.firestormviewer.FirestormViewer
+
+**Unofficial** Flatpak wrapper for Firestorm (https://www.firestormviewer.org/)
+
+### Broken
+* ~~Voice~~ *- Fixed*
+* ~~Discord~~ *- Fixed*
+
+### Build and Install
+Run this line by line.
+
+```shell
```

Comment:
Please don't duplicate flatpak's documentation here, especially given it's wrong. You don't need sudo to run flatpak-builder.

Reply:
That's my bad, that's from my personal repo where I was using that to install it to the system Flatpak install.

I've removed that section altogether.